server/bin/pbench-state-manager.py

```diff
@@ -119,20 +231,64 @@ def main(options):
         help="Specify owning user to create a new Dataset",
     )
     parser.add_argument(
-        "--path",
-        dest="path",
-        help="Specify a tarball filename (from which controller and name will be derived)",
+        "--did",
+        dest="did",
+        help=(
+            "Specify a completed sequence operator to be removed from the"
+            " pending set"
+        ),
+    )
+    parser.add_argument(
+        "--error", dest="error", help="Specify an error message to be recorded"
+    )
+    parser.add_argument(
+        "-g",
+        "--get-metadata",
+        action="append",
+        dest="get_metadata",
+        help="Print metadata values as key=value",
```

Comment:
I didn't really spend a lot of time finessing help strings: I don't expect anyone but me and my pal `pbench-copy-sosreports` to use it (and the later only until we get around to killing it), so at best they server as memory joggers; although it'd be almost as easy to read the source. But since I don't actually ever want to merge this PR, I'll spend some more time playing with them.